plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/sync/SyncVspCommand.java

```diff
@@ -39,4 +39,21 @@ public String getNuageVspEntity() {
         return _nuageVspEntity;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
This ``equals`` implementation should include attributes from the ``Command`` super class.